### PR TITLE
api: EditWiki implementation

### PR DESCRIPTION
**Pull request by m-pavel**
Proposed solution for the https://github.com/gogs/go-gogs-client/issues/110

### CODE REVIEW

File: internal/route/api/v1/api.go [hunk 1]
```diff
@@ -347,6 +346,7 @@ func RegisterRoutes(m *macaron.Macaron) {
 				}, reqRepoWriter())
 
 				m.Patch("/issue-tracker", reqRepoWriter(), bind(api.EditIssueTrackerOption{}), repo2.IssueTracker)
+				m.Post("/wiki", reqRepoWriter(), bind(api.EditWikiOption{}), repo2.Wiki)
```
Comment (unknwon) [inline internal/route/api/v1/api.go]:
```suggestion
				m.Patch("/wiki", reqRepoWriter(), bind(api.EditWikiOption{}), repo2.Wiki)
```

**Comment by unknwon** [comment]
Thanks, I've merged the dependent PR, I'm testing CI now.

**Comment by m-pavel** [comment]
Hi @unknwon, just realised that this PR has not been merged. I've merged master changes into this branch, so please approve if it's possible.

**Review by unknwon — APPROVED** [review]
Thanks for the followup!

**Pull request merged**